doc/example/store.ipynb

```diff
@@ -384,70 +553,45 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 11,
-   "metadata": {},
-   "outputs": [],
+   "execution_count": 12,
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "WARNING: You are loading a problem.\n",
+      "This problem is not to be used without a separately created objective.\n",
```

Comment:
Is this still correct or fixed in `develop`? #669

Reply:
it is still correct, as we do not save the objective but rather just information of the objective.